code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -160,6 +179,11 @@
 	. = ..()
 
 /mob/living/simple_animal/Life()
+	if(is_aquatic && !submerged() && stat != DEAD)
+		walk(src, 0)
+		if(PENDING_STATUS(src, STAT_PARA))
```

Comment:
Doesn't this mean it won't get paralyzed if it isn't already?

Reply:
But it already uses `SET_STATUS_MAX` anyway, why are we checking that to begin with...?